server/initialize.js

```diff
@@ -224,26 +225,22 @@ export function Initialize(server) {
         // Create the app registry file for the very first time
         saveConfiguration();
       } else {
-        // App registry file exists, just update it
-        const currentDate = new Date().toISOString();
-
         // If this function fails, it throws an exception
         const source = JSON.parse(fs.readFileSync(wazuhRegistry, 'utf8'));
 
         // Check if the stored revision differs from the package.json revision
         const isNewApp = packageJSON.revision !== source.revision;
```

Comment:
The name of this constant is ambiguous, you can change it to `isAppUpgraded`.

And also compare the version field of package.json